samwise.js

```diff
@@ -9,12 +9,13 @@
 ( function( document ) {
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.